src/main/java/org/terasology/tutorialWorldGenerationLakes/LakesProvider.java

```diff
@@ -38,19 +39,20 @@
 
     @Override
     public void setSeed(long seed) {
-        lakeNoise = new SubSampledNoise(new BrownianNoise(new PerlinNoise(seed + 3), 4), new Vector2f(0.001f, 0.001f), 1);
+        lakeNoise = new SubSampledNoise(new BrownianNoise(new PerlinNoise(seed + 3), 4), new Vector2f(0.001f, 0.001f)
+                , 1);
```

Comment:
That's an odd linebreak :thinking: Does it come from auto-formatting in IntelliJ? Sometimes you can help the formatter along with some manual linebreaks...

```java
        lakeNoise = new SubSampledNoise(
        		new BrownianNoise(new PerlinNoise(seed + 3), 4), 
        		new Vector2f(0.001f, 0.001f), 1);
```